ffi/capi/js/examples/tinywasm/delete_fetch.mjs

```diff
@@ -0,0 +1,9 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// diplomat-wasm.mjs uses fetch when available, but fetch in Node.js is broken,
```

Comment:
Do you want to file diplomat issue? If this a problem in one of our examples, it will also be a problem for clients

Reply:
Basically the same issue as https://github.com/rust-diplomat/diplomat/issues/283

Reply:
Link to that then?